cmd/process-agent/main_common.go

```diff
@@ -4,12 +4,15 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
nit: move to the the non stdlib imports

Reply:
I've seen this nit a few times now -- is there an automatic way to do so, or at least to check for it?